README.md

```diff
@@ -23,17 +23,17 @@ Config.php parameters to operate the server in master mode:
 
 ### Slave
 
-Config parameter to operate the server in slave mode:
+Config parameters to operate the server in slave mode:
 
 ````
-// can be chosen freely, you just have to make sure that the master and
-// all slaves have the same key. Also make sure to chose a strong shared secret.
+// can be chosen freely, you just have to make sure the master and
+// all slaves have the same key. Also make sure to choose a strong shared secret.
 'gss.jwt.key' => 'random-key',
 
 // operation mode
 'gss.mode' => 'slave',
 
-// url of the master, so that we can redirect the user back in case of an error
+// url of the master, so we can redirect the user back in-case of an error
```

Comment:
Actually `in case of` is an expression:
http://dict.leo.org/german-english/in%20case%20of

Reply:
Oops sorry, my mistake. Reject and I'll make all other tweaks again, or accept and I'll submit another in a moment to fix.. or option C someone with more GH experience can suggest :)